modules/jooby-hbv/src/main/java/io/jooby/hbv/HbvModule.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Jooby https://jooby.io
+ * Apache License Version 2.0 https://jooby.io/LICENSE.txt
+ * Copyright 2014 Edgar Espina
+ */
+package io.jooby.hbv;
+
+
+import edu.umd.cs.findbugs.annotations.NonNull;
+import io.jooby.Extension;
+import io.jooby.Jooby;
+import io.jooby.StatusCode;
+import io.jooby.validation.ConstraintViolationHandler;
+import jakarta.validation.ConstraintViolationException;
+import jakarta.validation.Validator;
+import jakarta.validation.ValidatorFactory;
+import org.hibernate.validator.HibernateValidator;
+import org.hibernate.validator.HibernateValidatorConfiguration;
+
+import java.util.function.Consumer;
+
+import static jakarta.validation.Validation.byProvider;
+
+public class HbvModule implements Extension {
```

Comment:
I do prefer: `HibernateValidatorModule` same for maven coordinates: `jooby-hibernate-validator`